ray-operator/config/samples/pytorch-text-classifier/ray-job-v1alpha1.pytorch-distributed-training.yaml

```diff
@@ -0,0 +1,41 @@
+# This RayJob is based on the "Fine-tune a PyTorch Lightning Text Classifier with Ray Data" example in the Ray documentation.
+# See https://docs.ray.io/en/master/train/examples/lightning/lightning_cola_advanced.html for more details.
+apiVersion: ray.io/v1alpha1
```

Comment:
@kevin85421 added a v1alpha1 since Kueue does not fully support v1 yet until  https://github.com/kubernetes-sigs/kueue/pull/1435 is merged